src/agent/config.ts

```diff
@@ -412,4 +427,6 @@ export const defaultConfig: ResolvedDebugAgentConfig = {
   forceNewAgent_: false,
   testMode_: false,
   resetV8DebuggerThreshold: 30,
+
+  useFirebase: false,
```

Comment:
If we were ever to make `firebase` the default, would we remove this toggle?

Reply:
Yes we would, and in the process would also retire a number of config values that would no longer apply.